driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -792,6 +807,81 @@ public void setWorkload(int node, Workload... workload) {
     execute(CCM_COMMAND + " node%d setworkload %s", node, workloadStr);
   }
 
+  private static VersionNumber getScyllaVersionThroughCcm(String versionString) {
+    try {
+      executeStatic("ccm create get_version -n 1 --scylla --version %s", versionString);
+      String versionOutput = executeStatic("ccm node1 versionfrombuild");
+      executeStatic("ccm remove");
```

Comment:
please:
1. Remove exact cluster
2. Move it to `finally`
3. Make it ignore if it fails

Reply:
Done